web-app/src/components/FeelingsSection.js

```diff
@@ -38,6 +38,14 @@ const Wrapper = styled.div`
   margin: 0 16.7%;
```

Comment:
We need to remove width: 50% here and make it maybe width 95% to match designs:

<img width="1007" alt="image" src="https://user-images.githubusercontent.com/20341285/109667881-813ce900-7b68-11eb-90f9-65f21b59aac9.png">